docs/examples/mcp-reverse-proxy/docker-compose.sse.yaml

```
@@ -0,0 +1,254 @@
+      timeout: 5s
+      retries: 5
+
+  # TRUF.NETWORK MCP Server with SSE transport
+  mcp-server:
+    image: trufnetwork/postgres-mcp:latest
+    container_name: truf-mcp-server
+    restart: unless-stopped
+    command: [
+      "--transport=sse",
+      "--sse-host=0.0.0.0",
+      "--sse-port=8000", 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)